apps.momits.Electrip.json

```diff
@@ -0,0 +1,142 @@
+{
+    "app-id": "apps.momits.Electrip",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.28",
```

Comment:
```suggestion
    "runtime-version": "3.30",
```

Reply:
With runtime version 3.30 the ```shared-modules``` build of Pygame fails on my PC.